lab-davide/server.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+
+//npm modules//
+const debug =require('debug')('note: server');
+const express  = require('express');
+const morgan = require('morgan');
+
+//app modules//
+const noteRouter = require('./route/note-router');
+const errorResponse = require('./lib/error-response');
+
+//env variables//
+const port = process.env.PORT || 3000;
+const app = express();
+
+
+// enabling middle ware//
+app.use(morgan('dev'));
```

Comment:
I usually attach middleware like bodyParser here too, just for readability, but the way you're doing it works too.